sympy/utilities/lambdify.py

```diff
@@ -450,6 +452,19 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def print_dummify(expr):
+    """
+    Returns the value of dummify to be used. This function either returns True or False.
+    >>> from sympy.utilities.lambdify import print_dummify
+    >>> from sympy.abc import x, y
+    >>> print_dummify(x + y**2)
```

Comment:
This input would never happen for this function. It would be better to use a Symbol for the example.